### PR TITLE
Add HsvColor.ToString() and TryParse() 

### PR DESCRIPTION
**Pull request by robloo**
## What does the pull request do?

HsvColor does not currently support serialization to strings, this PR enables it using a format similar to HTML/CSS.

## What is the current behavior?

No .ToString() or TryParse() support.

## What is the updated/expected behavior with this PR?

 * HsvColor.ToString() is implemented with a format similar to "hsva(230, 1.0, 0.5, 0.9)"
 * HsvColor.TryParse() is implemented supporting both "hsv(230, 1.0, 0.5)" and "hsva(230, 1.0, 0.5, 0.9)" formats (more forgiving).
 * HsvColor.Parse() is implemented internally using TryParse()
 * I noticed there is risk with Color.ToString() interpolation that the hex format could theoretically be incorrect. The issue is string interpolation uses the CurrentCulture instead of InvariantCulture by default. This hasn't been an issue because all cultures I can think of format hexadecimal numbers the same way. However, just to be theoretically correct, InvariantCulture is now used.
 * Wrap 360 degrees to 0 in the Hue component (since the PR is still open)

## How was the solution implemented (if it's not obvious)?

As stated.

## Checklist

- [ ] Added unit tests (if possible)?
- [X] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation

## Breaking changes

None

## Obsoletions / Deprecations

None

### CODE REVIEW

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0019551-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Comment by jmacato** [comment]
@robloo i wonder if you can add the CSS version of RGBA as well?

File: src/Avalonia.Visuals/Media/HsvColor.cs [hunk 1]
```diff
@@ -155,6 +157,109 @@ public Color ToRgb()
...
+
+            if (workingString.Length == 0 ||
+                workingString.IndexOf(",", StringComparison.Ordinal) < 0)
+            {
+                return false;
+            }
+
+            if (workingString.StartsWith("hsva(", StringComparison.OrdinalIgnoreCase) &&
+                workingString.EndsWith(")", StringComparison.Ordinal) &&
+                workingString.Length > 6)
```
Comment (maxkatz6) [inline src/Avalonia.Visuals/Media/HsvColor.cs:224]: nit: length checks should go first

Comment (robloo) [reply src/Avalonia.Visuals/Media/HsvColor.cs:224]: Yes, that's a good optimization

**Comment by robloo** [comment]
> @robloo i wonder if you can add the CSS version of RGBA as well?

I'll take a quick look. If it will be simple to add, sure.

File: src/Avalonia.Visuals/Media/HsvColor.cs (hunk 1 above)
Comment (robloo) [reply src/Avalonia.Visuals/Media/HsvColor.cs]: Updated

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0019554-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Comment by robloo** [comment]
@jmacato I added #7899. Let's not hold-up this PR and discuss all possible new changes in that issue. There is a lot of potential I think to extend this area.

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0019564-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0019574-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0019593-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Review by maxkatz6 — APPROVED** [review]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0019595-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Pull request merged**